package.json

```diff
@@ -43,7 +43,7 @@
     "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
-    "@saleor/app-sdk": "~0.9.1",
+    "@saleor/app-sdk": "0.10.0-2",
```

Comment:
Leaving comment here - only merge when sdk is on 1.10.0, lets remember not to depend on any rc versions

PS can we configure `release-it` to prefix rc like in CLI? (`0.10.0-rc.2` or something like that)?

Reply:
Yes, we can configure that. We'll do it after 0.10 to not switch patterns mid-way and confuse scripts.